File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        output = [0] * (len(nums) * 2)
        for i in range(len(nums)):
            output[i] = nums[i]
            output[len(nums)+i] = nums[i]
        return output

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        if (nums == null) {
            return 0;
        }

        Set<Integer> set = new HashSet<>();
        for (int num:nums) {
            set.add(num);
        }

        int output = 0;
        for (int num: set) {
            if (!set.contains(num-1)) {
                int len = 1;
                while (set.contains(num+len)) {
                    len++;
                }
                output = Math.max(output, len);

                if (output >= set.size()/2) {
                    return output;
                }
            }
        }
        return output;
    }
}
